challenge.rb

```diff
@@ -1,26 +1,32 @@
 
 def capitalize_each_string(input)
   #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
   #implement your solution here
+  input.reject{|e| e != "dog"}
 end
 
 def no_dogs_allowed(input)
   #implement your solution here
+  input.reject{|e| e== "dog"}
 end
 
 def count_the_animals(input)
   #implement your solution here
+  input.size
 end
 
 def fetch_the_first_two(input)
   #implement your solution here
+  input[0..1]
 end
 
 def fetch_CD_animals(input)
   #implement your solution here
+  input.reject{|e| !(["c", "d"].include? e[0])}
```

Comment:
You've already used reject in your fetch_the_dog method. Can you find another enumerable method to solve this?